extensions/iceberg/s3/src/main/java/io/deephaven/iceberg/util/IcebergToolsS3.java

```diff
@@ -62,4 +77,37 @@ public static IcebergCatalogAdapter createS3Rest(
         return new IcebergCatalogAdapter(catalog, fileIO);
     }
 
+    /**
+     * Create an Iceberg catalog adapter for an AWS Glue catalog. System defaults will be used to populate the region
+     * and credentials. These can be configured by following
+     * <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-authentication.html">AWS Authentication and
+     * access credentials</a> guide.
+     *
+     * @param name the name of the catalog; if omitted, the catalog URI will be used to generate a name
+     * @param catalogURI the URI of the AWS Glue catalog
+     * @param warehouseLocation the location of the S3 datafiles backing the catalog
+     * @return the Iceberg catalog adapter
+     */
+    public static IcebergCatalogAdapter createGlue(
+            @Nullable final String name,
+            @NotNull final String catalogURI,
+            @NotNull final String warehouseLocation) {
+
+        // Set up the properties map for the Iceberg catalog
+        final Map<String, String> properties = new HashMap<>();
+
+        final GlueCatalog catalog = new GlueCatalog();
+
+        properties.put(CatalogProperties.CATALOG_IMPL, catalog.getClass().getName());
+        properties.put(CatalogProperties.URI, catalogURI);
+        properties.put(CatalogProperties.WAREHOUSE_LOCATION, warehouseLocation);
+
+        // TODO: create a FileIO interface wrapping the Deephaven S3SeekableByteChannel/Provider
```

Comment:
Don't need duplicate todos. Make a ticket if you want to keep it. Consider rephrasing, since we don't necessarily plan to act on this yet.